GitTank/ViewModels/CreateBranchViewModel.cs

```diff
@@ -0,0 +1,62 @@
+﻿using GitTank.Loggers;
+using Microsoft.Toolkit.Mvvm.Input;
+using Microsoft.Extensions.Configuration;
+
+namespace GitTank.ViewModels
+{
+    public class CreateBranchViewModel : BaseViewModel
+    {
+        public delegate void OnClickEventHandler();
+        public event OnClickEventHandler OnClick;
```

Comment:
I was yesterday confused by this event name((
maybe something like CreateBranchesCompleted?